### DISCUSSION TITLE
odd issues when running wiki.js on page creation or edits

### DISCUSSION THREAD

**Discussion by mmd123** [requarks/wiki#3405] [Help / Questions]
**Question**
I'm not sure if this is a bug, issue, or configuration problem on my end, but its very odd, and yet it works still, when I go to create/edit a page, every single time I save the page or edit, IT SAVES STILL, but I get an eacces: permission denied error, before it successfully saves the changes and pages, so, it properly saves the stuff, but throws an error saying it cant still before doing so
**Host Info (please complete the following information):**
OS: Raspbian 10 (debian 10)
Wiki.js version: 2.4.107
Database engine: mariadb

**Comment by NGPixel** [comment]
Most likely a file permission issue with one of the storage module you enabled. If using the local disk module, make sure the wiki.js node process has write permissions to the folder you defined.
